.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
+    "arrowParens": "always",
+    "bracketSameLine": false,
```

Comment:
This seems to be an HTML/JSX formatting option, do we need/want it in JetStream?